mailmerge/api.py

```diff
@@ -94,6 +91,39 @@ def addattachments(message, template_path):
         original_text = message.get_payload()
         multipart_message.attach(email.mime.text.MIMEText(original_text))
         message = multipart_message
+    # HACK: For Python2 (see comments in `_create_boundary`)
+    message = _create_boundary(message)
+    return message
+
+
+def convert_markdown(message):
+    """Convert markdown in message text to HTML."""
+    if message.is_multipart():
+        print(">>> Warning: Markdown conversion requires non-multipart email.")
```

Comment:
Should this exit?

Reply:
If we choose to use the custom header option, then this condition would never be met. Resolving in b99e752345d41e33a43558f121e29d6f69ab792e.